src/Components/Articles/ArticleList.jsx

```diff
@@ -1,5 +1,26 @@
+import ArticleCard from "./ArticleCard";
+import { useState, useEffect } from "react";
+import { fetchArticles } from '../../apis'
 
 
 export default function ArticleList() {
-    
+    const [articleList, setArticleList] = useState([])
```

Comment:
Please include a loading state here and display a message to the user while the articles are loading.